runtime/src/chplmemtrack.c

```diff
@@ -76,6 +77,9 @@ static int hashSizes[NUM_HASH_SIZE_INDICES] = { 1543, 3079, 6151, 12289, 24593,
 static int hashSizeIndex = 0;
 static int hashSize = 0;
 
+static chpl_bool track_all_mds = false;
+#define DO_TRACK_MD(md) (chpl_mem_descTrack(md) || track_all_mds)
```

Comment:
Yeah, that's better from an encapsulation point of view. I'll do something like that.